Mage.Client/src/main/java/mage/client/dialog/GameEndDialog.java

```diff
@@ -104,9 +104,12 @@ private void saveGameLog(GameEndView gameEndView) {
                              .append(sdf.format(gameEndView.getStartTime()))
                              .append('_').append(gameEndView.getMatchView().getGameType())
                              .append('_').append(gameEndView.getMatchView().getGames().size())
-                             .append(".txt").toString();
+                             .append(".html").toString();
                      PrintWriter out = new PrintWriter(fileName);
-                     out.print(gamePanel.getGameLog());
+                     String log = gamePanel.getGameLog();
+                     log = log.replace("<body>", "<body style=\"background-color:black\">");
+                     log = log.replace("<font color=\"#CCCC33\">", "<br><font color=\"#CCCC33\">");
```

Comment:
Please add code comments about `CCCC33` color (that it is the color of the time and is made to separate the lines).

Reply:
Done!